psga/config/psga-schema-sars-cov-2-1.0.1.yaml

```diff
@@ -0,0 +1,476 @@
+        slug: midnight-ont-v2
+      sequencer_technology:
+        name: ont
+        slug: ont
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    # Midnight-ONT_V3
+    - name: illumina-sars-cov-2-midnight-ont-v13
+      kit:
+        name: Midnight-ONT_V3
+        slug: midnight-ont-v3
+      sequencer_technology:
+        name: illumina
+        slug: illumina
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    - name: ont-sars-cov-2-midnight-ont-v3
+      kit:
+        name: Midnight-ONT_V3
+        slug: midnight-ont-v3
+      sequencer_technology:
+        name: ont
+        slug: ont
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    # NEB-VarSkip_v1a
+    - name: illumina-sars-cov-2-neb-varskip-v1a
+      kit:
+        name: NEB-VarSkip_v1a
+        slug: neb-varskip-v1a
+      sequencer_technology:
+        name: illumina
+        slug: illumina
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    - name: ont-sars-cov-2-neb-varskip-v1a
+      kit:
+        name: NEB-VarSkip_v1a
+        slug: neb-varskip-v1a
+      sequencer_technology:
+        name: ont
+        slug: ont
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    # NEB-VarSkip_v1a-long
+    - name: illumina-sars-cov-2-neb-varskip-v1a-long
+      kit:
+        name: NEB-VarSkip_v1a-long
+        slug: neb-varskip-v1a-long
+      sequencer_technology:
+        name: illumina
+        slug: illumina
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    - name: ont-sars-cov-2-neb-varskip-v1a-long
+      kit:
+        name: NEB-VarSkip_v1a-long
+        slug: neb-varskip-v1a-long
+      sequencer_technology:
+        name: ont
+        slug: ont
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    # NEB-VarSkip_v2
+    - name: illumina-sars-cov-2-neb-varskip-v2
+      kit:
+        name: NEB-VarSkip_v2
+        slug: neb-varskip-v2
+      sequencer_technology:
+        name: illumina
+        slug: illumina
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    - name: ont-sars-cov-2-neb-varskip-v2
+      kit:
+        name: NEB-VarSkip_v2
+        slug: neb-varskip-v2
+      sequencer_technology:
+        name: ont
+        slug: ont
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    # NEB-VarSkip_v2b
+    - name: illumina-sars-cov-2-neb-varskip-v2b
+      kit:
+        name: NEB-VarSkip_v2b
+        slug: neb-varskip-v2b
+      sequencer_technology:
+        name: illumina
+        slug: illumina
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+    - name: ont-sars-cov-2-neb-varskip-v2b
+      kit:
+        name: NEB-VarSkip_v2b
+        slug: neb-varskip-v2b
+      sequencer_technology:
+        name: ont
+        slug: ont
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
+
+pipeline_version:
+  docker_tag: 1.0.1
```

Comment:
This is going to need to be templated...

Reply:
What do you have in mind when referring to "templated"?